### PR TITLE
Add Opta deploy to README.md

### PR DESCRIPTION
**Pull request by QuinnNgo**
# Summary
This PR adds a button that allows users to deploy `retool-onpremise` to AWS using a tool called [Opta](https://docs.opta.dev/). It works as follows:

1. The button links a user to a form, where the user fills out some configuration that Opta requires (similar to Heroku's deploy button). This configuration is used to generate an Opta configuration file, which is generated from [this template](https://github.com/tryretool/retool-onpremise/compare/master...QuinnNgo:master#diff-4e765eb002cca38a5fc67ff413c1f447f1838ca1af8d36fb3b1d654747e2fc4d).
2. The user downloads the generated configuration file and the Opta CLI, and then prepares their AWS credentials locally
3. The user deploys using the Opta CLI and the generated configuration file!

# Testing
**Note that the HTML for the button in this PR does not work yet, as it points to a file at the path https://github.com/tryretool/retool-onpremise/blob/master/opta/opta.yaml. This path will become valid after this PR is merged. In order to help in testing, I've added a button below which is equivalent, but points to the forked repo that this request is coming from:

[Deploy](https://app.runx.dev/deploy-with-aws?url=https%3A%2F%2Fgithub.com%2FQuinnNgo%2Fretool-onpremise%2Fblob%2Fmaster%2Fopta%2Fopta.yaml&name=Retool)

# Screenshots
image
image
image

### CODE REVIEW

**Review by NitinAgg — APPROVED** [review]

**Comment by QuinnNgo** [comment]
@kentwalters @gkunthara @markflorkowski Would love your opinions on adding this!

**Comment by markflorkowski** [comment]
@QuinnNgo 

I would move the opta.yml into a subdirectory `Opta` -- to match with the way that we are doing this with other services like Hekoku.

Additionally, would add an entry in the Table of Contents for the section that you are adding.

**Comment by markflorkowski** [comment]
hmm, actually I see that the `heroku.yml` is also at the root of the repo. Not sure if that should be the case 😂

File: README.md [hunk 1]
```diff
@@ -65,6 +65,14 @@ Spin up a new EC2 instance. If using AWS, use the following steps:
 1. Navigate to your server's IP address in a web browser. Retool should now be running on port `3000`.
 1. Click Sign Up, since we're starting from a clean slate. The first user to into an instance becomes the administrator.
 
+### Deploying to AWS using Opta
+
+Just use the Deploy to AWS button below!
+
+<a href="https://app.runx.dev/deploy-with-aws?url=https%3A%2F%2Fgithub.com%2Ftryretool%2Fretool-onpremise%2Fblob%2Fmaster%2Fopta.yml&name=Retool" target="_blank" rel="noopener noreferrer">
```
Comment (markflorkowski) [inline README.md:72]: Would do this in markdown 

```[Deploy](https://app.runx.dev/deploy-with-aws?url=https%3A%2F%2Fgithub.com%2Ftryretool%2Fretool-onpremise%2Fblob%2Fmaster%2Fopta.yml&name=Retool)```

**Review by markflorkowski — APPROVED** [review]

**Pull request merged**